client/dashboard/components/upsell-cta-button/index.tsx

```diff
@@ -7,24 +7,27 @@ import type { ComponentProps } from 'react';
 import './style.scss';
 
 type UpsellCTAButtonProps = ComponentProps< typeof Button > & {
-	trackId: string;
+	tracksId: string;
 	onClick?: ( event: React.MouseEvent< HTMLButtonElement | HTMLAnchorElement > ) => void;
 };
 
 export default function UpsellCTAButton( props: UpsellCTAButtonProps ) {
```

Comment:
@fushar I feel that this and

https://github.com/Automattic/wp-calypso/blob/497ffb10affa935c63bfc53fc05e570e484fdaa4/client/dashboard/sites/hosting-feature-gate/upsell.tsx#L14-L24

are a bit redundant, in that both have tracking baked-in. To explain further, now when the Hosting Feature gating upsell renders, it will trigger calypso_dashboard_hosting_feature_upsell_impression from `HostingFeatureUpsell` and calypso_dashboard_upsell_impression from `UpsellCTAButton`.

Reply:
Thinking about it some more, I'm now okay with us having two sets of upsell tracking when there's a callout + update CTA button for now. Ultimately, I don't think we should be aware of each other existence and decide which one to trigger, which adds mental cognition.

In the future, I can see a further abstraction of HostingFeatureGate to just FeatureGate, and remove the need for the UpsellCTAButton. But I think the current approach is good enough for now.

Reply:
Makes sense 👍